File: diet-plan/java_diet_regiment/Ingredient.java
class Ingredient {
    String name;
    double protein, calories;

    Ingredient(String ingredientName, double proteinContent, double caloriesContent) {

        this.name = ingredientName;
        protein = proteinContent;
        calories = caloriesContent;
    }

    public double getProtein() {
        return protein;
    }

    public double getCalories() {
        return calories;
    }
}

File: Ingredient.js
class Ingredient {
    constructor(ingredientName, proteinContent, caloriesContent) {
      this.name = null;
      this.protein = 0;
      this.calories = 0;
  
      this.name = ingredientName;
      this.protein = proteinContent;
      this.calories = caloriesContent;
    }

    getProtein() {
      return this.protein;
    }

    getCalories() {
      return this.calories;
    }
  }